Detections/SecurityEvent/PotenialResourceBasedConstrainedDelegationAbuse.yaml

```diff
@@ -0,0 +1,37 @@
+id: 2937bc6b-7cda-4fba-b452-ea43ba8e835f
+name:  Possible Resource-Based Constrained Delegation Abuse
+description: |
+  'This query identifies Active Directory computer objects modifications that allow an adversary to abuse the Resource-based constrained delegation. 
+  This query checks for event id 5136 that the Object Class field is "computer" and the LDAP Display Name is "msDS-AllowedToActOnBehalfOfOtherIdentity" which is an indicator of Resource-based constrained delegation.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1134
+query: |
+ SecurityEvent
+ | where EventID == 5136 
+ | parse EventData with * 'ObjectClass">' ObjectClass "<" *
+ | parse EventData with * 'AttributeLDAPDisplayName">' AttributeLDAPDisplayName "<" *
+ | parse EventData with * 'ObjectDN">' ObjectDN "<" *
+ | where ObjectClass == "computer" and AttributeLDAPDisplayName == "msDS-AllowedToActOnBehalfOfOtherIdentity"
```

Comment:
You could probably move the where condition above the  
| parse EventData with * 'ObjectDN">' ObjectDN "<" *      as we are not using the ObjectDN for comparison. 
This will make the query parse ObjectDN value only for those where the condition is matched making the query probably a little bit more performant.